### ISSUE TITLE
Error in rake telegram:bot:poller when using response typecasting

### ISSUE THREAD

**Issue by michaelkl** [telegram-bot-rb/telegram-bot#22]
Hi!
I'm using response typecasting. Following is my `/config/initializers/telegram_bot.rb` I use to set things up:
```
Telegram.bots_config = MyApp.config.fetch('telegram_bot', {}).symbolize_keys
Telegram::Bot::Client.typed_response!
```

And I got following error when running `rake telegram:bot:poller`:
```
$ bin/rake telegram:bot:poller
Running via Spring preloader in process 26085
Started bot poller.
no implicit conversion of String into Integer
/home/mike/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/telegram-bot-0.11.0/lib/telegram/bot/updates_poller.rb:68:in `[]'
/home/mike/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/telegram-bot-0.11.0/lib/telegram/bot/updates_poller.rb:68:in `fetch_updates'
/home/mike/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/telegram-bot-0.11.0/lib/telegram/bot/updates_poller.rb:48:in `start'
...
```
because it was waiting for `Hash`, not `Array` of `Telegram::Bot::Types::Update`.

**Comment by printercu** [comment]
Hi! Thank you!

Please try this patch locally (just add to your boot file):

```ruby
require 'telegram/bot/updates_poller'
module Telegram
  module Bot
    class UpdatesPoller
      def fetch_updates
        response = bot.async(false) { bot.get_updates(offset: offset, timeout: timeout) }
        updates = response.is_a?(Array) ? response : response['result']
        return unless updates && updates.any?
        reload! do
          updates.each do |update|
            @offset = update['update_id'] + 1
            yield update
          end
        end
      rescue Timeout::Error # rubocop:disable HandleExceptions
      end
    end
  end
end
```

I'll commit fix if everything is fine.

**Comment by michaelkl** [comment]
Yes, this patch fixes the issue!

**Comment by printercu** [comment]
Thank you! Fix is available in `0.11.1`.
